app/jobs/scrape_job.rb

```diff
@@ -0,0 +1,66 @@
+class PostScrapeWorker < ActiveJob::Base

```

Comment:
Have the file name and class name be in sync. Is there a name that reveals the purpose of this class a little clearer?